app/scripts/modules/core/diffs/commitHistory.component.ts

```diff
@@ -0,0 +1,34 @@
+import {module} from 'angular';
+
+import {IComponentOptions} from 'angular';
+
+export interface ICommit {
+  authorDisplayName: string;
+  commitUrl: string;
+  displayId: string;
+  id: string;
+  message: string;
+  timestamp: number;
+}
+
+class CommitHistoryComponent implements IComponentOptions {
+  public bindings: any = {
+    commits: '<'
+  };
+  public template = `
+  <div class="step-section-details" ng-if="$ctrl.commits && $ctrl.commits.length">
+    <table class="table table-condensed">
+      <tr><th>Date</th><th>Commit</th><th>Message</th><th>Author</th></tr>
+      <tr ng-repeat="commit in $ctrl.commits">
+        <td>{{commit.timestamp | date:'MM/dd'}}&nbsp;&nbsp;</td>
+        <td><a target="_blank" href="{{commit.commitUrl}}">{{commit.displayId}}</a>&nbsp;&nbsp;</td>
+        <td>{{commit.message | limitTo: 50}}&nbsp;&nbsp;</td>
```

Comment:
They were residual's from the original template and I suspect were for spacing when used in the pipeline execution details view.  I'm betting they can be safely removed.  Will check.